src/components/icon/svg/cc-logo.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think we need the SVGs, let's remove them

Reply:
I'd like to have them somewhere in source control just in case. Exporting them can be a pain.

Reply:
@emmawedekind added them in the invision board we have

Reply:
Hi yes, assets are here: https://projects.invisionapp.com/boards/UF3Q7E6QEZM/